lmnet/lmnet/datasets/base.py

```diff
@@ -159,6 +159,45 @@ def _change_gt_boxes_shape(self, gt_boxes_list):
         return np.array(results)
 
 
+class KeypointsDetectionBase(Base, metaclass=ABCMeta):
```

Comment:
On Google Scholar, "Key Point Detection" hits 5,340 documents, while "Key Points Detection" hits 917 documents. How about changing the class name to `KeypointDetectionBase`?

Also, according to the member method, this class exists for "single person pose estimation", hence "SinglePersonPoseEstimationBase" or something looks better class name.

Reply:
@tkng , thanks!

I have checked mscoco official site: http://cocodataset.org/#keypoints-2019
As you said, `KeypointDetectionBase` is better.

For `SinglePersonPoseEstimationBase` name, `crop_from_full_image()` can also be used for `top-down approach multi-person pose estimation` or `facial keypoint (landmark) detection`. (Although we won't provide them.) `KeypointDetectionBase ` should be better here I think.

Reply:
I see, thanks.